### ISSUE TITLE
clone event should clear pending games

### ISSUE THREAD

**Issue by oakmac** [oakmac/tourney-nerd#12]
When we clone an event, any teams should be cleared from Pending Games.

**Comment by oakmac** [comment]
Example:

```json
"game-cMEeozzWc3s3": {
  "id": "game-cMEeozzWc3s3",
  "name": "Finals Round 1: 6th vs 3rd",
  "scoreA": 0,
  "scoreB": 0,
  "status": "STATUS_SCHEDULED",
  "game-id": "game-cMEeozzWc3s3",
  "field-id": "field-tTzY73PenZeT",
  "group-id": "group-4TCxLwc2rKqP",
  "teamA-id": "team-aaaaaaaa",
  "teamB-id": "team-bbbbbbbb",
  "teamA-type": "TEAM_TYPE_DIRECT_TEAM",
  "teamB-type": "TEAM_TYPE_DIRECT_TEAM",
  "division-id": "division-KmbM3AMx8xe3",
  "timeslot-id": "timeslot-8zNxrhmWeNrR",
  "pending-teamA": {
    "type": "PENDING_GROUP_RESULT",
    "place": 6,
    "group-id": "group-DDqCeuYpvcXP"
  },
  "pending-teamB": {
    "type": "PENDING_GROUP_RESULT",
    "place": 3,
    "group-id": "group-DDqCeuYpvcXP"
  }
}
```

Should become:


```json
"game-cMEeozzWc3s3": {
  "id": "game-cMEeozzWc3s3",
  "name": "Finals Round 1: 6th vs 3rd",
  "scoreA": 0,
  "scoreB": 0,
  "status": "STATUS_SCHEDULED",
  "game-id": "game-cMEeozzWc3s3",
  "field-id": "field-tTzY73PenZeT",
  "group-id": "group-4TCxLwc2rKqP",
  "teamA-id": null,
  "teamB-id": null,
  "teamA-type": "TEAM_TYPE_DIRECT_TEAM",
  "teamB-type": "TEAM_TYPE_DIRECT_TEAM",
  "division-id": "division-KmbM3AMx8xe3",
  "timeslot-id": "timeslot-8zNxrhmWeNrR",
  "pending-teamA": {
    "type": "PENDING_GROUP_RESULT",
    "place": 6,
    "group-id": "group-DDqCeuYpvcXP"
  },
  "pending-teamB": {
    "type": "PENDING_GROUP_RESULT",
    "place": 3,
    "group-id": "group-DDqCeuYpvcXP"
  }
}
```

**Comment by oakmac** [comment]
This was annoying for the 2024 Woodlands Fall League. We should fix this for the next event.

**Comment by oakmac** [comment]
Fixed with [commit #a8d80197](https://github.com/oakmac/tourney-nerd/commit/a8d801974c850e82f0c1d987b2b324ef537f9a59)
